### ISSUE TITLE
Set up old "cheese grater" Mac

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#383]
Some where out of the way so it will we can evaluate use. Maybe in corner lab

**Comment by kaitlynrm** [comment]
I setup the computer with the other birds in 213 and will try to install ubuntu from a disk since it was unable to reboot from the USB.

**Comment by kaitlynrm** [comment]
We don't have a large enough disk to burn Ubuntu on. We only ahve 700 MB disks and the file is 1.51 MB.

**Comment by kubu4** [comment]
Looked a bit harder and found DVD-R in SAFS supply room. @kaitlynrm working on getting things ready now.

**Comment by kaitlynrm** [comment]
[I downloaded and burned 64-bit PC (AMD64) desktop image onto the disk,](http://releases.ubuntu.com/16.04.5/) Cheese grater isn't booting up from the CD drive. (You hold C down while rebooting for this option.) Instead, the Mac OS login screen comes up and the CD Drive ejects. [Here is a little more detail on what I have tried while attempting a boot from the disk.](https://wordpress.com/post/genefish.wordpress.com/3977)

I'll try to see if there is any other way to reformat a USB for Mac, but would love any other suggestions.
